project_event/views/calendar_event_category_view.xml

```diff
@@ -4,12 +4,16 @@
             model="ir.ui.view">
         <field name="name">calendar.event.form.inherit</field>
         <field name="model">calendar.event</field>
-        <field name="inherit_id" ref="calendar_resource.view_calendar_event_form" />
+        <field name="inherit_id" ref="calendar_resource.view_calendar_event_form"/>
         <field name="arch" type="xml">
             <xpath expr="//div[@class='oe_title']" position="after">
-                <label for="category_id" class="oe_edit_only" />
+                <label for="category_id" class="oe_edit_only"/>
                 <h2>
-                    <field name="category_id" attrs="{'readonly': [('is_task_event','=',True)], 'required': True}"/>
+                    <field name="category_id"/>
+                </h2>
+                <label for="sector_id" class="oe_edit_only"/>
+                <h2>
+                    <field name="sector_id"/>
```

Comment:
@jananjoy we need the same readonly attribute of the category in the sector_id field

Reply:
@rimbendhaou Done